src/graph/AdminExecutor.cpp

```diff
@@ -0,0 +1,129 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "http/HttpClient.h"
+#include "graph/AdminExecutor.h"
+#include "process/ProcessUtils.h"
+#include "webservice/Common.h"
+
+#include <folly/executors/Async.h>
+#include <folly/futures/Future.h>
+
+namespace nebula {
+namespace graph {
+
+AdminExecutor::AdminExecutor(Sentence *sentence,
+        ExecutionContext *ectx) : Executor(ectx) {
+    sentence_ = static_cast<AdminSentence*>(sentence);
+    header_["add_job"] = "Job Id";
+    header_["show_jobs"] = "Job Id,Command,Status,Start Time,Stop Time";
+    header_["show_job"] = "Job Id(TaskId),Command(Dest),Status,Start Time,Stop Time";
+    header_["stop_job"] = "Job Id";
+    header_["backup_job"] = "Job num backuped";
+    header_["recover_job"] = "Job num recovered";
```

Comment:
maybe header_{{"add_job": "Job id"}, {...}, {...}} is better

Reply:
thx, remove this map, use getHeader func instead.